Lib/xml/dom/minidom.py

```diff
@@ -32,6 +32,7 @@
 
 
 class Node(xml.dom.Node):
+    """Define properties accessible on a DOM node."""
```

Comment:
I think it’s strange to have a class docstring start with a verb; it makes sense for a method (that does something), but a class usually represents a thing (noun), it does not do something.

Reply:
Addressed in [b97d812](https://github.com/python/cpython/pull/128477/commits/b97d812e6dde095b004d72465ed3f8837e46b96c)